$NetBSD: patch-aa,v 1.3 2017/04/27 20:27:36 schmonz Exp $

Adjust to more modern Sys::Syslog. Fix typos in error messages.

--- greylist.orig	2005-01-17 08:38:07.000000000 -0500
+++ greylist
@@ -27,8 +27,9 @@ my $version = "0.3";
 
 # TODO: alternative logging options, instead of syslog?
 
-use Sys::Syslog;
+use Sys::Syslog qw(:DEFAULT setlogsock);
 
+setlogsock('unix');
 
 openlog "greylist", "pid", "mail";
 
@@ -338,14 +339,14 @@ cleanup_maybe;
 # TODO: change to an immediate error here..?
 unless ( defined $ENV{TCPREMOTEIP} )
 {
-	syslog "err", "ENV{TCPREMOEIP} not set!";
+	syslog "err", "ENV{TCPREMOTEIP} not set!";
 	smtp_temp_fail "Cannot find remote IP";
 }
 
 # now this should never happen...
 unless ( $ENV{TCPREMOTEIP} =~ m{^(\d+\.\d+\.\d+\.\d+)$} )
 {
-	syslog "err", "ENV{TCPREMOEIP} = $ENV{TCPREMOTEIP} - bad format!";
+	syslog "err", "ENV{TCPREMOTEIP} = $ENV{TCPREMOTEIP} - bad format!";
 	smtp_temp_fail "Bad format for remote IP";
 }
 
